packages/server/lib/modes/record.js

```diff
@@ -276,7 +276,10 @@ const uploadArtifactBatch = async (artifacts, protocolManager, quiet) => {
             url: artifact.uploadUrl,
             fileSize: artifact.fileSize,
             key: artifact.reportKey,
-            duration: performance.now() - startTime,
+            // performance.now() gives milliseconds with many decimals - this is
+            // higher resolution than we need, and the cy cloud column is int, not
+            // a floating poina
+            uploadDuration: Math.round(performance.now() - startTime),
```

Comment:
can we change this from duration to upload duration ? does this create an new column in the DB ?

Reply:
The new column has already been created in the DB, and the api schema has this field as `uploadDuration`